Src/MoneyFox.Core/Queries/Payments/GetPaymentsForAccountId/GetPaymentsForAccountIdQuery.cs

```diff
@@ -32,6 +34,8 @@ public class GetPaymentsForAccountIdQuery : IRequest<List<Payment>>
 
         public DateTime TimeRangeEnd { get; }
 
+        public int PaymentTypeFilter { get; set; }
```

Comment:
This shouldn't have a setter, since they're only supposed to be set via the constructor. The two existing of me are wrong as well so feel free to change those too :)

Reply:
Also I would use an enum here. Int's are much harder to understand what is meant. You could create a PaymentTypeFilter Enum with the default being "All".

Reply:
Got it. I'll remove all of the setters and switch from int to enum